README.md

```diff
@@ -151,7 +151,7 @@ Build and deploy the Spin app to Fermyon Cloud.
 | ------------- | -------- | --------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | fermyon_token | True     | -         | [Fermyon Cloud Personal Access Token](https://developer.fermyon.com/cloud/user-settings.md#create-and-manage-a-personal-access-token) for deploying the Spin app to Fermyon Cloud |
 | manifest_file | False    | spin.toml | Path to `spin.toml`.                                                                                                                                                              |
-
+| key_values    | False    | -         | Pass a key/value (key=value) to all components of the application. You can specify multiple key_values by putting each key/value pair on its own line. Refer to example below.    |
```

Comment:
Maybe change the description to "Pass one or more key/value pairs to the `default` store for all components"? Might want to call out which store name the key/values will end up in.